packages/babel-plugin-ima-logger/jest.config.js

```diff
@@ -0,0 +1,15 @@
+const base = require("../../jest.config.base.js");
+
+module.exports = {
+    ...base,
+    globals: {
+        functions: 10,
+        lines: 10,
+        statements: 10
+    },
+    testEnvironment: "node",

```

Comment:
This should go to the root config.

Reply:
One package has this value set on "jsdom", some packages haven't got this setting at all. Do we really want move this key to the root config?

Reply:
If one package has this option set to `jsdom` and actually needs it, then that single package should have this setting defined and override the default value. Well, to be honest, there is no need to set the `testEnvironment: "node"` at all, since it is default value, so we can actually remove it from all the packages, that are using it.

Reply:
Because setting`testEnvironment: "node"` is default, it's no more in config files.

Reply:
Some test's failed if there wasn't specified `testEnvironment: "node"`. So I added it into `jest.config.base.js`